_shells/tips-and-tricks.md

```diff
@@ -53,6 +53,14 @@ relative to the driver in the virtual environment location at:
 
 Under windows, [venv] will be located at _%programdata%\\qualisystems\\venv_.
 
+The Execution Server can be configured so that all drivers log to the same locaiton rather than under their own virtual environmaent location. Add the following key to the ES _customer.config_:
+
+{% highlight xml %}
+<add key="DefaultPythonEnvrionmentVariables" value="LOG_PATH=c:\[path]\[to]\[directory]"/>
```

Comment:
Depends on what you mean by "supposed to be". As we found out today, it only works if misspelled.  Looks like a bug against the execution server needs to be opened.